Add documentation for the LIFX Flame and Morph effects (#24331)
home-assistant/home-assistant.github.io: source/_integrations/lifx.markdown

          change: 35
```

### Hardware effects

> The Flame (`lifx_effect_flame`), Morph (`lifx.effect_morph`) and Move (`lifx.effect_move`) effects are hardware-based and only work on specific LIFX devices.  Flame and Morph are available on the LIFX Tile and Candle while the Move effect requires a LIFX Z, Lightstrip or Beam.

Reviewer: ```suggestion
The Flame (`lifx_effect_flame`), Morph (`lifx.effect_morph`) and Move (`lifx.effect_move`) effects are hardware-based and only work on specific LIFX devices. Flame and Morph are available on the LIFX Tile and Candle, while the Move effect requires a LIFX Z, Lightstrip, or Beam.
```

> All hardware-based effects can be stopped and started regardless of the power state of the device, but the default behavior for each service is turn the device on when starting an effect. Set the `power_on` attribute of the service to `false` to override this default.

Reviewer 2: ```suggestion
All hardware-based effects can be stopped and started regardless of the device's power state, but the default behavior for each service is to turn the device on when starting an effect. Set the `power_on` attribute of the service to `false` to override this default.
```

| `spread` | Maximum color difference between participating lights, in degrees on a color wheel (ranges from 0 to 359).
| `power_on` | Set this to False to skip the effect on lights that are turned off (defaults to True).

### Service `lifx.effect_flame`

> Run a hardware-based effect on LIFX matrix devices that creates a flame effects on the device. The speed of the flames is controlled by the `speed` attribute. The device will be powered on by default, but this can be overridden by setting `power_on` to `false`.

Reviewer 2: ```suggestion
Run a hardware-based effect on LIFX matrix devices that creates a flame effect on the device. The device will be powered on by default, but this can be overridden by setting `power_on` to `false`. The `speed` attribute controls the speed of the flames.
```

| Service data attribute | Description |
| ---------------------- | ----------- |
| `entity_id` | String or list of strings that point at `entity_id`s of matrix lights.
| `speed` | Duration in seconds for the effect to travel the length of the device (min: 1s, max: 25s)
| `power_on` | Whether to turn the light on before starting the effect (optional, default: true)

### Service `lifx.effect_morph`

> Run a hardware-based effect on LIFX matrix devices that animates blobs of colors across the device. The speed of the movement is controlled by the `speed` attribute.

Reviewer 2: ```suggestion
Run a hardware-based effect on LIFX matrix devices that animates blobs of colors across the device. The `speed` attribute controls the speed of the movement.
```

> You must provide either a `palette` or `theme` to use for the effect, but not both. The `palette` attribute allows you to select the colors used by the effect, while the `theme` attribute allows you to select one of the pre-configured themes, which match those found in the LIFX smart phone app.

Reviewer 2: ```suggestion
You must provide a `palette` or `theme` to use for the effect, but not both. The `palette` attribute allows you to select the colors used by the effect, while the `theme` attribute allows you to select one of the pre-configured themes which match those found in the LIFX smartphone app.
```

The device will be powered on by default, but this can be overridden by setting `power_on` to `false`.

| Service data attribute | Description |
| ---------------------- | ----------- |
| `entity_id` | String or list of strings that point at `entity_id`s of matrix lights.
| `speed` | Duration in seconds for the effect to travel the length of the device (min: 1s, max: 25s)
| `palette` | A list of at least 2 and at most 16 HSBK values to use for this effect.
| `theme` | The theme to use for the effect. Must be one of: `autumn`, `blissful`, `cheerful`, `dream`, `energizing`, `epic`, `exciting`, `focusing`, `halloween`, `hanukkah`, `holly`, `independence` `day`, `intense`, `mellow`, `peaceful`, `powerful`, `relaxing`, `santa`, `serene`, `soothing`, `sports`, `spring`, `tranquil`, `warming`.
| `power_on` | Whether to turn the light on before starting the effect (optional, default: true)

### Service `lifx.effect_move`

> Run a harware-based effect on LIFX multizone devices that a moves the colors currently set on the device in a particular direction. The direction and speed of the animation are controlled by the `speed` and `direction` attributes. You can change the colors of the effect while it is running using the `lifx.set_state` service.

Reviewer 2: ```suggestion
Run a hardware-based effect on LIFX multizone devices that move the current colors on the device in a particular direction. The direction and speed of the animation are controlled by the `speed` and `direction` attributes. You can change the effect's colors while running using the `lifx.set_state` service.
```
